embassy-usb/src/builder.rs

```diff
@@ -417,6 +417,10 @@ impl<'a, 'd, D: Driver<'d>> InterfaceAltBuilder<'a, 'd, D> {
         self.builder.config_descriptor.write(descriptor_type, descriptor);
     }
 
+    pub fn bos_capability(&mut self, capability_type: u8, capability: &[u8]) {
```

Comment:
This is the only piece I had to add to the builder. I'm not sure if this is the best spot to add it but I needed a way to add BOS capabilities within the interface API.